AmazonChimeSDK/AmazonChimeSDK/ingestion/IngestionEventConverter.swift

```diff
@@ -8,62 +8,79 @@
+        attributes[EAName.meetingErrorMessage.description] = AnyCodable(eventAttributes[EAName.meetingErrorMessage])
+        attributes[EAName.meetingStatus.description] = AnyCodable(meetingStatus)
+        attributes[EAName.poorConnectionCount.description] = AnyCodable(eventAttributes[EAName.poorConnectionCount])
+        attributes[EAName.retryCount.description] = AnyCodable(eventAttributes[EAName.retryCount])
+        attributes[EAName.videoInputError.description] = AnyCodable(videoErrorStr)
+        
+        meetingConfig?.metadataAttributes.forEach({ (key: String, value: Any) in
+            attributes[key] = AnyCodable(value)
+        })
+        
         // Some meta data like meetingId is needed
         // Since these meta data changes from meeting to meeting
         return IngestionMeetingEvent(name: String(describing: event.name),
-                                     eventAttributes: IngestionEventAttributes(timestampMs: eventAttributes[EventAttributeName.timestampMs] as? Int64,
-                                                                               maxVideoTileCount: eventAttributes[EventAttributeName.maxVideoTileCount] as? Int,
-                                                                               meetingStartDurationMs: eventAttributes[EventAttributeName.meetingStartDurationMs] as? Int64,
-                                                                               meetingDurationMs: eventAttributes[EventAttributeName.meetingDurationMs] as? Int64,
-                                                                               meetingErrorMessage:
-                                                                               eventAttributes[EventAttributeName.meetingErrorMessage] as? String,
-                                                                               meetingStatus: meetingStatus,
-                                                                               poorConnectionCount: eventAttributes[EventAttributeName.poorConnectionCount] as? Int,
-                                                                               retryCount: eventAttributes[EventAttributeName.retryCount] as? Int,
-                                                                               videoInputError: videoErrorStr,
-                                                                               meetingId: meetingConfig?.meetingId,
-                                                                               attendeeId: meetingConfig?.attendeeId))
+                                     eventAttributes: attributes)
     }
 
     func toIngestionRecord(dirtyMeetingEvents: [DirtyMeetingEventItem], ingestionConfiguration: IngestionConfiguration) -> IngestionRecord {
         if dirtyMeetingEvents.isEmpty {
-            return IngestionRecord(metadata: IngestionMetadata(), events: [])
+            return IngestionRecord(metadata: [:], events: [])
```

Comment:
could you explain?

Reply:
It's just creating an empty dict since metadata type is changed from `IngestionMeatadata` to `Dict`.